TESTING.md

```diff
@@ -161,27 +161,22 @@ search engine.
 ### Test 1: CFIA Logo Redirect Functionality
 
 **Objective:** Verify that clicking the CFIA logo redirects the user to the
-application's base path as defined by the `VITE_BASENAME` environment variable.
+application's base path `/`.
```

Comment:
I'm confused as to why we are removing the basename functionality? This was for the cases was not served at the root of the domain

Reply:
@k-allagbe

Reply:
I removed it as it was no longer used. If we ever need it again, we could always put it back.